cdap-cli/src/main/java/co/cask/cdap/cli/command/HelpCommand.java

```diff
@@ -67,4 +73,93 @@ public String getPattern() {
   public String getDescription() {
     return "Prints this helper text";
   }
+
+  /**
+   * Prints the given command pattern which wrap at the given column width. It prints on multiple lines like this:
```

Comment:
Prints the given command pattern with text wrapping at the given column width. It prints multiple lines as: